src/Kafka/CreateKafkaInstance/components/CloudProviderTiles.tsx

```diff
@@ -86,6 +86,7 @@ export const CloudProvidersTiles: VoidFunctionComponent<
 };
 
 const tiles: { [id in CloudProvider]: JSX.Element } = {
-  aws: <AwsIcon size="lg" color="black" />,
-  azure: <MicrosoftIcon size="lg" color="black" />,
+  aws: <img src={LogoAWS} alt={""} style={{ height: "60px" }} />,
+  gcp: <img src={LogoGCP} alt={""} style={{ height: "60px" }} />,
+  azure: <img src={LogoAzure} alt={""} style={{ height: "60px" }} />,
```

Comment:
I feel like these images would make a great candidate to become components in the shared folder, so to use them like `<AWSLogo />` or something like that